### ISSUE TITLE
ocserv: pending rebuild for http-parser

### ISSUE THREAD

**Issue by MingcongBai** [AOSC-Dev/aosc-os-abbs#1939]
Bug description
-----------------------

With current `ocserv v0.12.2`, the package is missing a dynamic shared object from `http-parser`.

```
ocserv: error while loading shared libraries: libhttp_parser.so.2.8: cannot open shared object file: No such file or directory
``` 

Reproducing steps
---------------------------

Install `ocserv` and execute `ocserv`.

Update channel(s)
--------------------------

Stable

Package and version
------------------------------

`ocserv` v0.12.2

**Comment by MingcongBai** [comment]
Mischaracterised issue.

**Comment by MingcongBai** [comment]
Addressed with #2882. Closing.
